docs/basicusage.rst

```diff
@@ -38,6 +36,10 @@ Next, add your sources and operators. For easy testing, we'll use an :ref:`RSS
     url = http://blog.inquest.net/atom.xml
     feed_type = messy
 
+.. _operator:
+Similarly the operators are identify a name, a module, and a file for output.  The module specifies the format of the output.
```

Comment:
Operators aren't guaranteed to use a file - sqs for example accepts unlimited arbitrary config keys: http://threatingestor.readthedocs.io/en/latest/plugins/operator.html#amazon-sqs. Maybe just "...and additional configuration"?